lib/matplotlib/tests/test_path.py

```diff
@@ -77,9 +78,33 @@ def test_exact_extents(path, extents):
     # the way out to the control points.
     # Note that counterintuitively, path.get_extents() returns a Bbox, so we
     # have to get that Bbox's `.extents`.
+    path, extents = precomputed_curve.path, precomputed_curve.extents
     assert np.all(path.get_extents().extents == extents)
 
 
+@pytest.mark.parametrize('precomputed_curve', _test_curves)
+def test_signed_area(precomputed_curve):
+    path, area = precomputed_curve.path, precomputed_curve.area
+    assert np.isclose(path.signed_area(), area)
+    # now flip direction, sign of *signed_area* should flip
+    rverts = path.vertices[:0:-1]
+    rverts = np.append(rverts, np.atleast_2d(path.vertices[0]), axis=0)
+    rcurve = Path(rverts, path.codes)
+    assert np.isclose(rcurve.signed_area(), -area)
+
+
+def test_signed_area_unit_rectangle():
+    rect = Path.unit_rectangle()
+    assert np.isclose(rect.signed_area(), 1)
+
+
+def test_signed_area_unit_circle():
+    circ = Path.unit_circle()
+    # not quite pi, since it's not a "real" circle, just an approximation of a
+    # circle made out of bezier curves
+    assert np.isclose(circ.signed_area(), 3.1415935732517166)
```

Comment:
Seems like this number would pass with `np.pi`...

Reply:
moved the "exact" result to the comment